crates/uv-cli/src/lib.rs

```diff
@@ -2635,6 +2635,10 @@ pub struct BuildArgs {
     #[arg(long, conflicts_with = "list")]
     pub force_pep517: bool,
 
+    /// Remove build artifacts from the output directory after a successful build.
```

Comment:
I think this needs to be clearer that it'll remove _stale_ build artifacts and not the new ones :)